submissions/the-darkesttt/js-dom/script.js

```diff
@@ -0,0 +1,78 @@
+    },
+    {
+        title: 'Jupiter',
+        desc: "Jupiter is the fifth planet from the Sun and the largest in the Solar System. It is a gas giant with a mass more than two and a half times that of all the other planets in the Solar System combined, but slightly less than one-thousandth the mass of the Sun. Jupiter is the third brightest natural object in the Earth's night sky after the Moon and Venus, and it has been observed since prehistoric times. It was named after the Roman god Jupiter, the king of the gods.",
+    },
+    {
+        title: 'Saturn',
+        desc: "Saturn is the sixth planet from the Sun and the second-largest in the Solar System, after Jupiter. It is a gas giant with an average radius of about nine and a half times that of Earth. It has only one-eighth the average density of Earth; however, with its larger volume, Saturn is over 95 times more massive",
+    },
+    {
+        title: 'Uranus',
+        desc: "Uranus is the seventh planet from the Sun. Its name is a reference to the Greek god of the sky, Uranus, who, according to Greek mythology, was the great-grandfather of Ares (Mars), grandfather of Zeus (Jupiter) and father of Cronus (Saturn). It has the third-largest planetary radius and fourth-largest planetary mass in the Solar System. Uranus is similar in composition to Neptune, and both have bulk chemical compositions which differ from that of the larger gas giants Jupiter and Saturn. ",
+    },
+    {
+        title: 'Neptune',
+        desc: "Neptune is the eighth planet from the Sun and the farthest known solar planet. In the Solar System, it is the fourth-largest planet by diameter, the third-most-massive planet, and the densest giant planet. It is 17 times the mass of Earth, and slightly more massive than its near-twin Uranus. Neptune is denser and physically smaller than Uranus because its greater mass causes more gravitational compression of its atmosphere. It is referred to as one of the solar system's two ice giant planets (the other one being Uranus)."
+    },
+];
+
+const solarTitle = document.querySelector('.wrapper__title');
+const asideList = document.querySelector('.aside__list');
+const solarSystemTitle = document.querySelector('.container__title');
+const solarSystemDesc = document.querySelector('.container__desc');
+
+solarTitle.addEventListener('click', function() {
+    solarSystemTitle.innerText = solarSystem.title;
+    solarSystemDesc.innerText = solarSystem.desc;
+
+    if (selected) {
+        selected.classList.remove('focused');
+    }
+});
+
+let selected;
+
+asideList.addEventListener('click', function(event) {
+    const listTarget = event.target;
+    if (listTarget.tagName != 'LI') return;
+    highlightFocused(listTarget);
+
+    const thisTitle = listTarget.innerText;
+
+    const thisInPlanets = planets.filter(item => {
+        return item["title"] === thisTitle;
```

Comment:
```suggestion
        return item.title === thisTitle;
```

Redundant array notation